src/helpers/api/apiAjax.js

```diff
@@ -15,14 +15,12 @@ class ApiAjax {
         const url = formatUrl(path);
 
         req.onload = () => {
-          if (req.status === 500) {
-            reject(req.response);
-            return;
+          if (req.status !== 200) {
```

Comment:
This is super bad 😭 

![image](https://cloud.githubusercontent.com/assets/49499/19020839/bfb8f194-88b3-11e6-831e-e655de4ff414.png)